### PR TITLE
update GH Actions

### PR DESCRIPTION
**Pull request by Jason2866**
and create tagged release

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -46,19 +46,19 @@ jobs:
       - name: Run PyInstaller
         run: |
           python -m PyInstaller tdmgr32.spec
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v3
         with:
-          name: Windows_tdmgr32
+          name: app
```
Comment (jziolkowski) [inline .github/workflows/main.yml:51]: Why this change? Just curious

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -102,9 +102,31 @@ jobs:
...
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: app
+          path: ./dist
+      - name: setenv release tag
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Release
+        uses: jason2866/action-gh-release@v1.1
+        #if: startsWith(github.ref, 'refs/tags/')
```
Comment (jziolkowski) [inline .github/workflows/main.yml:126]: Pls remove comment

**Review by jziolkowski — CHANGES_REQUESTED** [review]

**Comment by jziolkowski** [comment]
Cool stuff with automatic release 👍👍👍

File: .github/workflows/main.yml (hunk 1 above)
Comment (Jason2866) [reply .github/workflows/main.yml:51]: We need the same path for uploaded artifacts. The name of the apps is not changed.

File: .github/workflows/main.yml (hunk 2 above)
Comment (Jason2866) [reply .github/workflows/main.yml]: done
